static/js/themeSwitcher.js

```diff
@@ -1,6 +1,6 @@
+/* eslint-disable no-var */
```

Comment:
Why?

Reply:
Because we only use `var`s. And this file wrongfully used const. Either we do it consistently everywhere and drop old IE support later, or we just stick to what we did.